test/content-parser.test.js

```diff
@@ -391,10 +384,10 @@ test('remove all should remove all existing parsers and reset cache', t => {
   contentTypeParser.getParser('application/xml') // fill cache with one entry
   contentTypeParser.removeAll()
 
-  t.same(contentTypeParser.cache.size, 0)
-  t.same(contentTypeParser.parserList.length, 0)
-  t.same(contentTypeParser.parserRegExpList.length, 0)
-  t.same(Object.keys(contentTypeParser.customParsers).length, 0)
+  t.assert.deepStrictEqual(contentTypeParser.cache.size, 0)
+  t.assert.deepStrictEqual(contentTypeParser.parserList.length, 0)
+  t.assert.deepStrictEqual(contentTypeParser.parserRegExpList.length, 0)
+  t.assert.deepStrictEqual(Object.keys(contentTypeParser.customParsers).length, 0)
```

Comment:
testing length here i think `strictEqual` should be used
```suggestion
  t.assert.strictEqual(Object.keys(contentTypeParser.customParsers).length, 0)
```

Reply:
You're right, I'll change that...
This was `t.same` before, which is equivalent to `deepStrictEqual`, but `strictEqual` is better.